### PR TITLE
chore: add close inactive issues workflow

### PR DESCRIPTION
**Pull request by zhyncs**
## Motivation

When an issue has not been updated for more than a month, mark it as inactive and close it.

cc @merrymercy @Ying1123 @hnyls2002 

I have tested it in my personal repository and there were no problems, but I am not sure if the repositories under the organization are also ok.

## Modification

add GitHub workflow

## Checklist

1. Ensure pre-commit `pre-commit run --all-files` or other linting tools are used to fix potential lint issues.
2. Confirm that modifications are covered by complete unit tests. If not, please add more unit tests for correctness.
3. Modify documentation as needed, such as docstrings or example tutorials.

### CODE REVIEW

**Review by Ying1123 — APPROVED** [review]

File: .github/workflows/close-inactive-issues.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
           script: |
-            const thirtyDaysAgo = new Date(Date.now() - 30 * 24 * 60 * 60 * 1000);
+            const thirtyDaysAgo = new Date(Date.now() - 60 * 24 * 60 * 60 * 1000);
```
Comment (Ying1123) [inline .github/workflows/close-inactive-issues.yml]: -> sixtyDaysAgo

File: .github/workflows/close-inactive-issues.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                  const issues = await fetchIssues(page);
+                  console.log(`Fetched ${issues.data.length} issues on page ${page}`);
+                  
+                  if (issues.data.length === 0) {
+                    hasMoreIssues = false;
+                    break;
+                  }
+                  
+                  for (const issue of issues.data) {
+                    if (new Date(issue.updated_at) < thirtyDaysAgo) {
```
Comment (Ying1123) [inline .github/workflows/close-inactive-issues.yml]:
```suggestion
                    if (new Date(issue.updated_at) < sixtyDaysAgo) {
```

File: .github/workflows/close-inactive-issues.yml (hunk 1 above)
Comment (zhyncs) [reply .github/workflows/close-inactive-issues.yml]: ok

**Pull request merged**